CONTRIBUTING.md

```diff
@@ -131,15 +132,15 @@ CSS
 
 ```
 
-Here is a full table of available CSS variables:
+Here are all the available CSS variables:
 | Variable | Description | Use |
 |---|---|---|
 | **`--darkreader-neutral-background`** | Neutral background color that <br>corresponds to the user's settings. | Mostly used for elements that have <br>a wrong background color |
 | **`--darkreader-neutral-text`** | Neutral text color that <br>corresponds to the user's settings. | Used for elements with a wrong text color |
 | **`--darkreader-selection-background`** | The text color setting <br>defined by the user. | The user's Text Color setting |
 | **`--darkreader-selection-text`** | The background color setting <br>defined by the user. | The user's Background Color setting |
 
-## Fixes for Filter and Filter+ mode
+## Fixes for Filter & Filter+ modes
```

Comment:
```suggestion
## Fixes for Filter & Filter+ mode
```

Modes implies that there are more a-like modes.

Reply:
Aren't Filter and Filter+ separate modes?  They are in the UI.

Reply:
They are, but if you both mention them you don't need to include the `s`, if you only mention `Filter` then you can use `modes`. At least, that's how I remember my English classes, and it does read better that way.